### ISSUE TITLE
Itemkey not suported

### ISSUE THREAD

**Issue by Gmatoy** [kyuknis/Zabbix_MSMQ_Monitor#1]
I did carefully all the steps 3 times in 3 diferent servers, did some alterations and tests and rolled back to default, still, the itemkey always became unsupported, can you help me? This script's proupose is awesome and i would love to implement im my environment.

**Comment by Gmatoy** [comment]
PS: ah testing in 2 different zabbix's , v3.0.2 and v3.2.7

**Comment by kyuknis** [comment]
Correct, I'll be adding this fact to the documentation soon. We're you able to discover a fix? I'll be looking into it tomorrow evening.

**Comment by Gmatoy** [comment]
No, i can't make scritps or progamming, but this installation was very alike, if not almost identical of the monitoring lld IIS site and pool (only thing that changes is the scripts and the template of course) . So i am sure that i installed correctly.
If u fix this, can i spread it on a zabbiz brazilian community? we are very active and this would make this scripts less underground e underrated.
This script ideia is awesome.
here is a print:
![image](https://user-images.githubusercontent.com/29662882/33429122-9e6428ea-d5b2-11e7-800f-c83f0e4d9973.png)

**Comment by Gmatoy** [comment]
If u fix this, can i spread it on a zabbiz brazilian community? we are very active and this would make this scripts less underground e underrated.
This script ideia is awesome.

**Comment by Gmatoy** [comment]
![image](https://user-images.githubusercontent.com/29662882/33429122-9e6428ea-d5b2-11e7-800f-c83f0e4d9973.png)

**Comment by kyuknis** [comment]
Hey, yes the IIS Site/AppPool monitoring is where a lot of this was learned so it is going to look similar. I fully intend to take a look at it, but the time for me to do so has moved to this Saturday because it involves booting up a couple of VM's for testing/development. I have to do the things that pay the bills first :). And so far as spreading it on the community, I already plan on putting it on share.zabbix.com, however you are free to distribute it as much as you like, just remember to be responsible in how it's shared and maintain the GPLv3 license. To see more about what you're allowed to do with it, go to https://github.com/kyuknis/Zabbix_MSMQ_Monitor/blob/master/LICENSE

**Comment by kyuknis** [comment]
@Gmatoy Could you please delete the template that you imported into Zabbix and import the new one I just uploaded?

**Comment by Gmatoy** [comment]
I know how it is man, chills!
the spread is all about just giving the link of your own github =D
i will se the permissions configuration of powershell for the agent and the scritps, and use your new template.
Many tanks!

**Comment by kyuknis** [comment]
All's good, glad to have helped! Hope all's well! God bless!
